test/api.test.js

```diff
@@ -20,6 +19,7 @@ describe('API', () => {
   };
   const compiler = {
     outputPath: '/output',
+    getInfrastructureLogger: () => console,
```

Comment:
Just mock, here https://github.com/webpack/webpack-dev-middleware/pull/494 this solved